### PR TITLE
[PPP-5166] - Use of Vulnerable Component: Components/jackson-databind 2.14.2 (Non OSGi upgrade)

### PR DESCRIPTION
**Pull request by soagarwal1**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 58s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox
```
---
:ok_hand: All tests passed!
 
Tests run: **12**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/metadata-editor/job/PR-286/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: pom.xml [hunk 1]
```diff
@@ -250,6 +250,20 @@
       <groupId>pentaho-kettle</groupId>
       <artifactId>kettle-engine</artifactId>
       <version>${pdi.version}</version>
+      <exclusions>
```
Comment (NJtwentyone) [inline pom.xml:253]: Is the jackson-databind going to be in lib folder ?

The jira descriptions scope of work has conflicting text

```
pme-ee/metadata-editor/libext/pentaho/jackson-databind-2.14.2.jar Pentaho Metadata Editor is bringing in the jackson-databind dependency through kettle-core. It needs to be excluded as there is no usecase for this library in PME
This needs to be updated as mentioned in the next to next point

pme-ee/metadata-editor/libext/pentaho/jackson-databind-2.14.2.jar this is the dependency for pme we need to upgrade to 2.16.2 over here
```
ppp_5166_pme_jackson-databind_description

**Review by NJtwentyone — CHANGES_REQUESTED** [review]
Might be more of a clarifying scope of work, see comment https://github.com/pentaho/pentaho-metadata-editor/pull/286/files#r1765172748.

File: pom.xml (hunk 1 above)
Comment (soagarwal1) [reply pom.xml:253]: I had clarified this with Ramaiz earlier and had made changes to the description. But later we found out that it was not needed and could be excluded. This discussion was done on a call. 

Jackson-databind will not be in the lib folder, aslo jackson-annotations and jackson-core will not be there

Comment (NJtwentyone) [reply pom.xml:253]: yeah just update the scope. I would have a final comment with the final decisions on all.

**Review by NJtwentyone — APPROVED** [review]

**Pull request merged**